Product/SoapUI_Test/RegressionSuite/Standard/AuditLogging-Standard/RetrieveDocumentAuditLog.properties

```diff
@@ -43,7 +43,7 @@ ActiveParticipant_Dest.@UserID=http://www.w3.org/2005/08/addressing/anonymous
 ActiveParticipant_Dest.@UserIsRequestor=true
 ActiveParticipant_Dest.@UserName=Karl Skagerberg
 ActiveParticipant_Dest.@AlternativeUserID=19233@192.168.1.2
-ActiveParticipant_Dest.@NetworkAccessPointID=localhost
+ActiveParticipant_Dest.@NetworkAccessPointID=localhost                                           
```

Comment:
This is a lot of trailing space, does this affect anything or does SoapUI trim the values?  Either way, we should clean it for consistency.